From 6e53d78136dff5f5bb395c2cc4ec93d2fbc0b1a0 Mon Sep 17 00:00:00 2001
From: Valerio Pilo <vpilo@coldshock.net>
Date: Fri, 15 Feb 2019 23:01:59 +0100
Subject: [plasma-pa] Increase minimum size of expanded plasmoid

Summary:
When outside of the tray, the plasmoid felt really cramped. This makes it more bearable.

Before: {F6615039}
After: {F6615040}

Reviewers: #plasma, broulik, drosca, ngraham

Reviewed By: ngraham

Subscribers: abetts, ngraham, plasma-devel

Tags: #plasma

Differential Revision: https://phabricator.kde.org/D19013
---
 applet/contents/ui/main.qml | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/applet/contents/ui/main.qml b/applet/contents/ui/main.qml
index 8ae841a..6d8d77c 100644
--- a/applet/contents/ui/main.qml
+++ b/applet/contents/ui/main.qml
@@ -39,15 +39,15 @@ Item {
     property string displayName: i18n("Audio Volume")
     property QtObject draggedStream: null
 
-    Layout.minimumHeight: units.gridUnit * 12
-    Layout.minimumWidth: units.gridUnit * 12
+    Layout.minimumHeight: units.gridUnit * 18
+    Layout.minimumWidth: units.gridUnit * 18
     Layout.preferredHeight: units.gridUnit * 20
     Layout.preferredWidth: units.gridUnit * 20
 
     Plasmoid.icon: paSinkModel.preferredSink && !isDummyOutput(paSinkModel.preferredSink) ? Icon.name(paSinkModel.preferredSink.volume, paSinkModel.preferredSink.muted)
                                                                                           : Icon.name(0, true)
-    Plasmoid.switchWidth: units.gridUnit * 12
-    Plasmoid.switchHeight: units.gridUnit * 12
+    Plasmoid.switchWidth: units.gridUnit * 18
+    Plasmoid.switchHeight: units.gridUnit * 18
     Plasmoid.toolTipMainText: {
         var sink = paSinkModel.preferredSink;
         if (!sink) {
@@ -291,6 +291,8 @@ Item {
 
     Plasmoid.fullRepresentation: ColumnLayout {
         spacing: units.smallSpacing
+        Layout.minimumHeight: main.Layout.minimumHeight
+        Layout.minimumWidth: main.Layout.minimumWidth
 
         function beginMoveStream(type, stream) {
             if (type == "sink") {
-- 
cgit v1.1

